data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/HistogramAggregateAction.java

```diff
@@ -101,6 +110,19 @@ private double convertToDouble(Number value) {
         return doubleValue;
     }
 
+    public Exemplar createExemplar(final String id, final Event event, double value) {
+        long curTimeNanos = getTimeNanos(Instant.now());
+        Map<String, Object> attributes = event.toMap();
+        if (Objects.nonNull(id)) {
+            attributes.put("exemplar_id", id);
```

Comment:
Is this the right naming convention? Does this work with the OpenSearch schemas?

Reply:
I changed it to `exemplarId`